src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -163,7 +166,7 @@ public void Disconnect()
         /// with the currently connected account
         /// </summary>
         /// <returns>Team projects associated with current account or null if disconnected</returns>
-        public IEnumerable<IProject> GetTeamProjects()
```

Comment:
>IProject [](start = 27, length = 8)

Is this still the public interface that gets called by the app? If so, then I'd argue that we should be using the generic types defined in the app-visible interface